### ISSUE TITLE
Corrupt video/hung after blood flood cutscene in 1st level of RoE

### ISSUE THREAD

**Issue by KozGit** [KozGit/DOOM-3-BFG-VR#82]
Starting a new game in RoE, after the player gets the artifact, the next brief cutscene is when the player walks down the hall a few steps, the hall floods with blood and screen turns red.

Screen never fades out of red, and player is unable to continue past this point.

Edit: Thought this may have been due to the changes I made to the fullscreen FX, but looks like this started with your most recent pull.  Maybe something in the flicksync code hanging after that particular cutscene?  I'll try to look at it later, need to go before I fall asleep at the keyboard.

**Comment by CarlKenner** [comment]
That cutscene is now called CUTSCENE_BLOOD, BTW.

I made some changes to screen fades in cutscenes, but that should just make them invisible when you need to see the HUD in a Flicksync.

I guess it must be pausing at the end of the cutscene. But I don't know why. That cutscene has no dialog, so it shouldn't ever be paused, even in Flicksync mode. I'll have to take a look.

**Comment by CarlKenner** [comment]
Found the bug. At first I thought it was actually caused by Flicksync_ResumeCutscene() being called after the end of the first cutscene (ironically to make sure we prevent issues like this).

When I fixed support for our Flicksync character having multiple lines in a row, I added a check to ResumeCutscene so that if the line we are paused for is by the same character as our Flicksync character, then we set the line we were paused for to be the new line we're supposed to say. But I forgot to check that we were actually paused, or that there was actually a paused line, or that the character wasn't NONE. So I'm telling it to wait for an empty (or garbage) line.

But then I noticed I was clearing any waiting lines at the start of the next cutscene... BUT I also call Flicksync_ResumeCutscene after that at the start of the cutscene, which is what is setting the waiting line.

I'll fix it now.
